htdocs/core/lib/pdf.lib.php

```diff
@@ -2262,7 +2262,7 @@ function pdf_getTotalQty($object, $type, $outputlangs)
  * 	@param	Translate	$outputlangs	Object lang for output
  * 	@return	array                       Linked objects
  */
-function pdf_getLinkedObjects($object, $outputlangs)
+function pdf_getLinkedObjects(&$object, $outputlangs)
```

Comment:
Why do you need the  &  ? No reason to modify object in a read only method.

Reply:
because i use $object->note_public to print order ref if there're too much linked orders

Reply:
@eldy